cime_config/testdefs/testmods_dirs/clm/run_self_tests/shell_commands

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 ./xmlchange CLM_FORCE_COLDSTART="on"
 
-# We use this testmod in a _Ln1 test; this requires forcing the ROF coupling frequency to every time step
-./xmlchange ROF_NCPL=48
+# We use this testmod in a _Ln1 test; this requires forcing the ROF coupling frequency to same frequency as DATM
+./xmlchange ROF_NCPL='$ATM_NCPL'
+
+# Turn MEGAN off to run faster
+./xmlchange CLM_BLDNML_OPTS='--no-megan' -append
+
+# Use fast structure and NWP configuration for speed
+./xmlchange CLM_STRUCTURE="fast"
+./xmlchange CLM_CONFIGURATION="nwp"
+
+# Turn cpl history off
+./xmlchagne HIST_OPTION="never"
```

Comment:
These changes do change answers, I didn't catch this when I pushed this.

SMS_D_Ln1.f10_f10_mg37.I2000Clm50BgcCropQianRs.derecho_intel.clm-run_self_tests.GC.ctsm5361erikb4bdelist

For this type of test it doesn't matter though.